api/src/main/java/org/apache/iceberg/DataOperations.java

```diff
@@ -44,6 +44,13 @@ private DataOperations() {
    */
   public static final String REPLACE = "replace";
 
+  /**
+   * Files are removed, replaced or modified, while changing the data in the table.
+   * <p>
+   * This operation is implemented by {@link ModifyFiles}.
+   */
+  public static final String MODIFY = "modify";
```

Comment:
Files are never modified, so I think this is really an overwrite: files were added to replace files that were deleted. The set of rows in the table changed, unlike "replace" that implies the set of rows did not change. I think the two are essentially the same thing, we just need to make sure it is clear that both these uses are covered by overwrite.

Reply:
Agree